Gemfile

```diff
@@ -105,3 +106,8 @@ gem 'active_storage_validations'
 
 # cài đặt imagemagick để sử dụng 2 gem dưới
```

Comment:
PLEASE don't use any Vietnamese words in the project !!!

Reply:
nhiều quá e ko sửa được sau e sẽ viết lại bằng tiếng a